### ISSUE TITLE
Is menu ready for stable yet?

### ISSUE THREAD

**Issue by westphaldp** [jeaye/ncurses-rs#183]
I tried to get it to work with Rust stable for days and had to find out that it's unfortunately not possible right now.

All menu library functions are behind a feature gate meaning that ncurses-rs as it is right now still compiles with Rust stable and won't break for anyone. It's like an opt-in extra feature for those using rust nightly.

But if you really don't want any unstable code in ncurses-rs you may also wait until the CString reform lands in Rust stable.

_Originally posted by @tsurai in https://github.com/jeaye/ncurses-rs/pull/87#issuecomment-114729666_

Being new to Rust, I can't be sure, but it looks like CString is stable now. Can we remove the nightly requirement?

**Comment by tsurai** [comment]
You are right. I've done a short test and everything seems to work fine when compiling it with stable. Should be ok to remove the notice saying that it requires nightly at this point.
